arbnode/espresso.go

```diff
@@ -0,0 +1,28 @@
+package arbnode
+
+import (
+	"github.com/offchainlabs/nitro/arbos/arbostypes"
+	"github.com/offchainlabs/nitro/arbutil"
+	"github.com/offchainlabs/nitro/execution"
+	"github.com/offchainlabs/nitro/util/containers"
+)
+
+// TransactionStreamerExecutionSequencer defines the interface under which
+// the TransactionStreamer interacts with the ExecutionSequencer interface.
+//
+// It derives the method definitions from the execution package:
+// "github.com/offchainlabs/nitro/execution"
+//
+// It is defined separately here from the execution.ExecutionSequencer
```

Comment:
This comment doesnt match the actual interface: https://github.com/OffchainLabs/nitro/blob/v3.6.7/execution/interface.go#L64-L73

Reply:
This is true.

This was **very recently** modified to be `ExecutionClient` instead of `ExecutionSequencer`.  So when I rebased, I missed that the i[nterface being utilized was changed.](https://github.com/EspressoSystems/nitro-espresso-integration/commit/e4bdf8121a35cf097c5c947a15246218b02a61af#diff-4c43892baa92d7cf1b422d5e9e7d5534ca1109dbecd5b647c8f8aaf57d3cc391R60)

Reply:
Actually, double checking this.  It seems like the `ExecutionClient` is a [requirement](https://github.com/OffchainLabs/nitro/blob/v3.6.7/execution/interface.go#L65) of `ExecutionSequencer`.  So this statement is still technically true.  But it would be better served to be more specific.